.github/workflows/tests.yml

```
@@ -8,7 +8,7 @@ on:
 
 env:
   TF_VERSION: 2.4
-  TORCH_VERSION: 1.7
+  TORCH_VERSION: 1.9.0

```

```suggestion
  TORCH_VERSION: 1.8.0
```

Can we try running the CI checks with `1.8.0` instead? I'm wondering if the problems are arising from the torch version.

---

@albi3ro it was on 1.8 originally, I bumped it up for the same reason (to see if it would fix the CI). No 🎲 :(

---

```suggestion
  TORCH_VERSION: 1.9.0+cpu
```

This will work! It was installing the `rocm` specific version, which was in beta.